src/test/java/it/pagopa/pn/deliverypush/utils/HtmlSanitizerTest.java

```diff
@@ -151,25 +150,21 @@ void sanitizeRequestAcceptedTemplateWithImgHTMlElement() {
-        assertThat(sanitizedTemplateModel).isNotEqualTo(templateModel);
-        assertThat(sanitizedTemplateModel).isInstanceOf(Map.class);
-
         Map<String, Object> sanitizedTemplateModelMap = (Map<String, Object>) sanitizedTemplateModel;
         NotificationInt sanitizedNotificationInt = (NotificationInt) sanitizedTemplateModelMap.get(FIELD_NOTIFICATION);
 
-        assertThat(sanitizedNotificationInt.getRecipients().get(0).getDenomination()).doesNotContain("<h1>", "<img");
-        assertThat(sanitizedNotificationInt.getIun()).isEqualTo(notificationInt.getIun());
-        assertThat(sanitizedNotificationInt.getSender()).isEqualTo(notificationInt.getSender());
-        assertThat(sanitizedNotificationInt.getAmount()).isEqualTo(notificationInt.getAmount());
-        assertThat(sanitizedNotificationInt.getPaProtocolNumber()).isEqualTo(notificationInt.getPaProtocolNumber());
-        assertThat(sanitizedNotificationInt.getSubject()).isEqualTo(notificationInt.getSubject());
-        assertThat(sanitizedNotificationInt.getSentAt()).isEqualTo(notificationInt.getSentAt());
-        assertThat(sanitizedNotificationInt.getDocuments()).isEqualTo(notificationInt.getDocuments());
-
-        assertThat(sanitizedTemplateModelMap.get(FIELD_SEND_DATE)).isEqualTo(templateModel.get(FIELD_SEND_DATE));
-        assertThat(sanitizedTemplateModelMap.get(FIELD_SEND_DATE_NO_TIME)).isEqualTo(templateModel.get(FIELD_SEND_DATE_NO_TIME));
-        assertThat(sanitizedTemplateModelMap.get(FIELD_DIGESTS)).isEqualTo(templateModel.get(FIELD_DIGESTS));
-        assertThat(sanitizedTemplateModelMap.get(FIELD_ADDRESS_WRITER)).isNotNull().isInstanceOf(PhysicalAddressWriter.class);
+        Assertions.assertAll(
+                () -> Assertions.assertEquals(sanitizedNotificationInt.getIun(), notificationInt.getIun()),
+                () -> Assertions.assertEquals(sanitizedNotificationInt.getSender(), notificationInt.getSender()),
+                () -> Assertions.assertEquals(sanitizedNotificationInt.getAmount(), notificationInt.getAmount()),
+                () -> Assertions.assertEquals(sanitizedNotificationInt.getPaProtocolNumber(), notificationInt.getPaProtocolNumber()),
+                () -> Assertions.assertEquals(sanitizedNotificationInt.getSubject(), notificationInt.getSubject()),
+                () -> Assertions.assertEquals(sanitizedNotificationInt.getSentAt(), notificationInt.getSentAt()),
+                () -> Assertions.assertEquals(sanitizedNotificationInt.getDocuments(), notificationInt.getDocuments()),
+                () -> Assertions.assertEquals(sanitizedTemplateModelMap.get(FIELD_SEND_DATE), templateModel.get(FIELD_SEND_DATE)),
+                () -> Assertions.assertEquals(sanitizedTemplateModelMap.get(FIELD_SEND_DATE_NO_TIME), templateModel.get(FIELD_SEND_DATE_NO_TIME)),
+                () -> Assertions.assertEquals(sanitizedTemplateModelMap.get(FIELD_DIGESTS), templateModel.get(FIELD_DIGESTS))
+        );
```

Comment:
Negli assert all vedo meno elementi dei precedenti assertThat quali sono stati eliminati e perchè?

Reply:
@cFiorilloMediatica lascerei le asserzioni fatte con assertJ che sono più parlanti rispetto a quelle di default di jUnit. Correggerei i codeSmells nel modo indicato da SonarCloud, ad esempio "Join these multiple assertions subject to one assertion chain":

BEFORE:
assertThat(someList).hasSize(3);
assertThat(someList).contains("something");

AFTER:
assertThat(someList)
  .hasSize(3)
  .contains("something");